tests/main/views/test_services.py

```diff
@@ -209,14 +209,14 @@ def test_enabled_service_has_unavailable_banner(self):
         self.service = self._get_g6_service_fixture_data()
 
         self.service['services']['status'] = 'enabled'
-        self.service['serviceMadeUnavailableAuditEvent'] = \
-            self._get_status_update_audit_event_for(
-                update_type='update_service_status',
-                old_status='published',
-                new_status='enabled',
-                timestamp='2016-01-05T17:01:07.649587Z',
-                service=self.service['services']
-            )
+        audit_event = self._get_status_update_audit_event_for(
+            update_type='update_service_status',
+            old_status='published',
+            new_status='enabled',
+            timestamp='2016-01-05T17:01:07.649587Z',
+            service=self.service['services']
+        )
+        self.service['serviceMadeUnavailableAuditEvent'] = audit_event
```

Comment:
Yep, getting rid of the line continuation is nice, but feels like the assignment can still all be done in one step.